linter.py

```diff
@@ -16,14 +16,13 @@
 class HamlLint(RubyLinter):
     """Provides an interface to haml-lint."""
 
-    syntax = 'ruby haml'
-    cmd = 'ruby -S haml-lint'
+    cmd = 'ruby -S haml-lint ${args} ${file}'
```

Comment:
I think you must use `${temp_file}` here. 😁 

(If you think that this is somewhat an stupid API, feel free to suggest something. FYI, We actually *need* explicit `cmd`s so that users can just e.g. do `cmd = 'eslint .'` or `cmd = 'golint $file_path'` etc.)